compose/compose-file/index.md

```diff
@@ -1662,6 +1662,9 @@ the service's containers.
   Default value is world-readable permissions (mode `0444`).
   The writable bit MUST be ignored if set. The executable bit MAY be set.
 
+> **Note** The `uid`, `gid` and `mode` options do not work in non-Swarm mode, i.e.
```

Comment:
@glours @ulyssessouza Could you confirm if this is correct?

Reply:
Well, I opened this issue because I found those not to be working (see also the discussion linked to in the PR), and people were complaining that this was not documented :)

But would be good if someone can double-check.

Reply:
Just a intermediate update, the team is looking into this and should have a more relevant update soon. Thanks for your follow-up on this.